config.env.template

```diff
@@ -32,6 +32,8 @@ P0_PAGERDUTY_ALERT_EVENTS_URL=
 P1_PAGERDUTY_INTEGRATION_KEY=
 P1_PAGERDUTY_ALERT_EVENTS_URL=
 
+SNS_TOPIC_ARN=
```

Comment:
I feel like we should update the documentation to reflect that an SNS routing feature is now supported

Reply:
[655294b](https://github.com/base-org/pessimism/pull/200/commits/655294b1b10c0d11ed7a34b5f42d87540a31c7e6)